generate/efi/acpidump/Makefile

```diff
@@ -44,7 +44,9 @@ OBJECTS = \
 	$(OBJDIR)/utmath.o\
 	$(OBJDIR)/utprint.o\
 	$(OBJDIR)/utstring.o\
-	$(OBJDIR)/utxferror.o
+	$(OBJDIR)/utxferror.o\
+	${OBJDIR}/utnonansi.o\
+	${OBJDIR}/utascii.o
 
```

Comment:
We have a big EFI fix series here:
https://github.com/acpica/acpica/pull/149
This has already been fixed in this big series.
No need to include this into this commit.
It is a separate issue.
Please refer to:
https://github.com/acpica/acpica/pull/149/commits/7786e436

Please drop this change block from this commit.